lib/phoenix/router.ex

```diff
@@ -28,17 +28,72 @@ defmodule Phoenix.Router do
         get "/pages/:page", PageController, :show
       end
 
-  The `get/3` macro above accepts a request of format `"/pages/VALUE"` and
-  dispatches it to the show action in the `PageController`.
-
-  Routes can also match glob-like patterns, routing any path with a common
-  base to the same controller. For example:
-
-      get "/dynamic*anything", DynamicController, :show
+  The `get/3` macro above accepts a request to `/pages/hello` and dispatches
+  it to the `PageController`'s `show` action with `%{"page" => "hello"}` in
+  `params`.
 
   Phoenix's router is extremely efficient, as it relies on Elixir
   pattern matching for matching routes and serving requests.
 
+  ## Routing
+
+  `get/3`, `post/3`, `put/3` and other macros named after HTTP verbs are used
+  to create routes.
+
+  The route:
+
+      get "/pages", PageController, :index
+
+  matches a `GET` request to `/pages` and dispatches it to the `index` action in
+  `PageController`.
+
+      get "/pages/:page", PageController, :show
+
+  matches `/pages/hello` and dispatches to the `show` action with
+  `%{"path" => "hello"}` in `params`.
```

Comment:
@jayjun Should it not match `%{"page" => "hello"}` in `params`, as `:page` is the param name?

Reply:
@jarroput Good spot, feel free to open a PR.

Reply:
Good catch @jarroput! Fix in #3526.